var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -61,6 +61,8 @@ class Emacs(AutotoolsPackage, GNUMirrorPackage):
     conflicts('@:26.3', when='platform=darwin os=catalina')
     conflicts('+native', when='@:27', msg="native compilation require @master")
 
+    patch('glibc-2.34.patch')
```

Comment:
Can you copy the comment in the Gentoo patch over the directive? That will give people context by reading only `package.py`. Also is the patch to be applied unconditionally (i.e. is it valid for each version of emacs we support)?